src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -18,8 +19,16 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            BuildWebHost(args)
-                .RunAsync()
+            var host = BuildWebHost(args);
+
+            var environment = host.Services.GetService<IEnvironment>();
+            if (environment.IsLinuxContainerEnvironment())
+            {
+                // Linux containers always start out in placeholder mode
+                environment.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "1");
+            }
+
```

Comment:
@fabiocav -- I moved this up here. I suppose we could just modify the Environment variables directly at this point, since no one would realistically be replacing IEnvironment and hitting it from here. That's the way it was originally, but I didn't like it b/c it was ignoring our abstraction.